tez/README.MD

```diff
@@ -5,20 +5,21 @@ This initialization action installs the latest version of [Apache Tez](https://t
 1. Required libraries such as [`protobuf`](https://github.com/google/protobuf) are installed
 2. Tez is compiled
 3. Tez is put into HDFS
-4. Hadoop is configured to work with Tez on the Dataproc cluster
+4. Hadoop and Hive are configured to work with Tez on the Dataproc cluster
 
 ## Using this initialization action
 You can use this initialization action to create a new Dataproc cluster with Apache Tez installed by:
 
 1. Uploading a copy of this initialization action (`tez.sh`) to [Google Cloud Storage](https://cloud.google.com/storage).
 1. Using the `gcloud` command to create a new cluster with this initialization action. **Note** - you will need to increase the `initialization-action-timeout` to at least 15 minutes to allow for Tez and its dependencies to compile and install. The following command will create a new cluster named `<CLUSTER_NAME>`, specify the initialization action stored in `<GCS_BUCKET>`, and increase the timeout to 15 minutes.
-   
+
     ```bash
     gcloud dataproc clusters create <CLUSTER_NAME> \
-    --initialization-actions gs://<GCS_BUCKET>/tez.sh   
+    --initialization-actions gs://<GCS_BUCKET>/tez.sh
```

Comment:
Needs a \ at the end, or we can maybe just remove the timeout override here; from my experience it finishes fast enough.